### PR TITLE
Added Dockerfile using JRE 16 for nigthlies #dockersupport-5

### PR DESCRIPTION
**Pull request by hantu85**
I've prepared `Dockerfile` and updated the readme file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ Tigase's Docker package for the Tigase XMPP Server.
 
 ## Simple tags
 -   [`nightly` (*nightly/Dockerfile*)](nightly/Dockerfile)
+-   [`nightly-jre16` (*nightly/Dockerfile*)](nightly/Dockerfile)
```
Comment (woj-tek) [inline README.md]: Shouldn't the path here points to `nightly/jre-16/Dockerfile`?

File: nightly/jre-16/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,39 @@
+FROM adoptopenjdk/openjdk16:debianslim-jre
```
Comment (woj-tek) [inline nightly/jre-16/Dockerfile:1]: What was the motivation to switch from `openjdk` repository? lack of "jre-slim" version?

File: README.md (hunk 1 above)
Comment (hantu85) [reply README.md]: Yes, you are correct, I'll fix it.

File: nightly/jre-16/Dockerfile (hunk 2 above)
Comment (hantu85) [reply nightly/jre-16/Dockerfile:1]: I found following statement that suggested to look elsewhere:
> Starting with openjdk:12 the default image as well as the -oracle and -oraclelinux7 variants are based on the official Oracle Linux 7 image which is provided under the GPLv2 as per the Oracle Linux End User Agreement (EULA).
> 
> The OpenJDK binaries in the default image as well as the -oracle and -oraclelinux7 variants are built by Oracle and are sourced from the OpenJDK community. These binaries are licensed under the GPLv2 with the Classpath Exception.

see https://hub.docker.com/_/openjdk?tab=description&page=1&ordering=last_updated&name=jre

Comment (woj-tek) [reply nightly/jre-16/Dockerfile:1]: Understood, makes sense.

**Pull request merged**